index.js

```diff
@@ -7,22 +7,44 @@ function init(){
     .then(drinkData => drinkData.drinks.forEach(renderCocktails))
 }
 
+function initialFocus(){
+    fetch(baseUrl + '/random.php')
+    .then(res => res.json())
+    .then(drinkData => {
+        const drink = drinkData.drinks
+        renderMain(drink)
+    })
+}
+
 function renderCocktails(drinks){
     const card = document.createElement('div')
     card.id = 'drink_card'
 
     const name = document.createElement('h2')
     name.textContent = drinks.strDrink
-    console.log(name)
+    // console.log(name)
 
     const image = document.createElement('img')
     image.src = drinks.strDrinkThumb
     image.details = drinks
-    console.log(image.details)
+    // console.log(image.details)
```

Comment:
going to approve this, but in the future I think we should not push console log's to the main branch.